api/server/server.go

```diff
@@ -740,6 +740,15 @@ func (s *Server) postImagesCreate(eng *engine.Engine, version version.Version, w
 		}
 	}
 
+	var (
+		opErr   error
+		useJSON = version.GreaterThan("1.0")
+	)
+
+	if useJSON {
```

Comment:
I was actually more wondering if we needed it because I think the other server code will set the content-type for us.
But if so we can always remove it later on if needed.